packages/rlc-common/src/interfaces.ts

```diff
@@ -172,7 +172,7 @@ export type PathParameter = {
 
 export interface OperationHelperDetail {
   lroDetails?: OperationLroDetail;
-  isPageable?: boolean;
+  isPaging?: boolean;
```

Comment:
why we need this change ?

Reply:
it's the property name suggested by cspell.